lib/src/realm_object.dart

```diff
@@ -313,7 +313,10 @@ mixin RealmObjectBase on RealmEntity implements RealmObjectBaseMarker, Finalizab
   };
 
   /// @nodoc
-  static Object? get<T extends Object?>(RealmObjectBase object, String name) {
+  static Object? get<T extends Object?>(RealmObjectBase object, String name, {bool validateBacklink = false}) {
```

Comment:
I agree, that it is not good we to change the interface, that's why it was added as a discussion point. 
An optional parameter wouldn't be a problem, but still this parameter is too specific for such a general method.
Unfortunately, we don't have any information about `RealmPropertyType` and this is the reason I have added this parameter.

Reply:
it is not easy to generalize it and if we do it it will negatively impact the benefit of doing it. Hence I suggested to check in-place in the generated code

Reply:
Ok. This is done.